Force a re-render to update recipe

mwcraig opened the pull request:
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed) (N/A build 1 never built)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

mwcraig committed:
Empty commit for re-render

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

mwcraig commented:
@conda-forge-admin, please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-build 3.18.2, conda-smithy 3.3.4, and conda-forge-pinning 2019.05.31

mwcraig committed:
Patch setup to avoid helpers bootstrap downloading

mwcraig committed:
Change version to 1.3.0.post1

1.3.0 was only released as a zip and, I'm guessing, had Windows line endings
too.

mwcraig committed:
Fix checksum

Why did this build locally?!

mwcraig committed:
Fix patch to use correct option to suppress downloading
